plugin/trino-exchange-filesystem/src/test/java/io/trino/plugin/exchange/filesystem/AbstractTestExchangeManager.java

```diff
@@ -88,7 +89,26 @@ public void testHappyPath()
             throws Exception
     {
         ExchangeId exchangeId = createRandomExchangeId();
-        Exchange exchange = exchangeManager.createExchange(new ExchangeContext(new QueryId("query"), exchangeId), 2, false);
+        Exchange exchange = exchangeManager.createExchange(
+                new ExchangeContext() {
```

Comment:
Please create `TestingExchangeContext` and reuse.

Reply:
done